Gemfile.lock

```diff
@@ -189,7 +189,7 @@ GEM
     smart_properties (1.15.0)
     sorbet (0.5.6357)
       sorbet-static (= 0.5.6357)
-    sorbet-runtime (0.5.6357)
+    sorbet-runtime (0.5.6361)
```

Comment:
Can we not comment this change? Not that it makes a difference for the release, but it would be great if the versions of runtime and static were in sync.

Reply:
Just saw that. Removing now